packages/opentelemetry-core/src/baggage/propagation/W3CBaggagePropagator.ts

```diff
@@ -58,7 +58,8 @@ export class W3CBaggagePropagator implements TextMapPropagator {
   }
 
   extract(context: Context, carrier: unknown, getter: TextMapGetter): Context {
-    const headerValue: string = getter.get(carrier, BAGGAGE_HEADER) as string;
+    const headerBaggage = getter.get(carrier, BAGGAGE_HEADER);
+    const headerValue: string = (Array.isArray(headerBaggage))? headerBaggage.join(','): headerBaggage as string;
```

Comment:
style nit

```suggestion
    const headerValue: string = Array.isArray(headerBaggage) ? headerBaggage.join(',') : headerBaggage as string;
```
I think you might also not need the type assertion? Not sure why it was there before.

```suggestion
    const headerValue: string = Array.isArray(headerBaggage) ? headerBaggage.join(',') : headerBaggage;
```

Reply:
@dyladan fixed the styling and removed the assertion. the line after checks if undefined so we are safe